File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        l,r=0,len(numbers)-1

        while(l<r):
            sum=numbers[r]+numbers[l]
            if sum>target:
                r=r-1
            elif sum<target:
                l=l+1
            else:
                return [l+1,r+1]
        return []

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {

        int l=0,r=numbers.length-1;
        int sum=0;
        while(l<r){
            sum=numbers[l]+numbers[r];
            if(sum>target)
            {
                r--;
                continue;
            }
            else if(sum<target){
                l++;
                continue;
            }
            else{
                return new int[]{l+1,r+1};
            }
        }
        return new int[]{};
    }
}